data/json/recipes/electronics/tools.json

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "result": "makeshift_oscilloscope",
+    "type" : "recipe",
+    "category": "CSC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_TOOLS",
+    "skill_used": "electronics",
+    "difficulty": 4,
+    "skills_required": [ "fabrication", 2 ],
+    "time": 240000,
+    "book_learn": [ [ "advanced_electronics", 4 ], [ "mag_electronics", 6 ] , [ "manual_electronics", 5 ] ],
```

Comment:
I quite like the idea of not having access to a tool forcing alternate strategy (in this case dismantling electrical items for parts you are unable to craft). That said `MULTIMETER 1` is reserved for a good example of an improvised multimeter and I'm open to suggestions as to this?